src/infinite-list/index.html

```diff
@@ -1,17 +1,17 @@
 <ul ref="wrapper" class="gb-list__wrapper {isGrid ? 'gb-list__wrapper--grid' : 'gb-list__wrapper--column'}">
-  <li data-is="gb-list-item" class="load first-loader" if="{$infinite.isFetchingBackward}" _consumes="infinite">
-    <gb-infinite-loader label="{$infinite.loaderLabel}"></gb-infinite-loader>
+  <li data-is="gb-list-item" class="load first-loader" if="{props.isFetchingBackward}">
```

Comment:
`loaderLabel` needs to be passed down as a prop. `gb-list-item` is a component (note the `data-is`), which creates a new scope.